apps/authentication-service/src/resources/clients/clients.test.ts

```diff
@@ -0,0 +1,36 @@
+import { Client } from "auth0";
```

Comment:
Typically a test file has the same name as the file it's testing. I can't tell which file this is testing.